WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/RestoreActivity.kt

```diff
@@ -113,7 +111,7 @@ class RestoreActivity : LocaleAwareActivity() {
             it.applyIfNotHandled {
                 when (this) {
                     is VisitSite -> {
-                        // todo annmarie add to ActivityLauncher
+                        ActivityLauncher.openUrlExternal(this@RestoreActivity, url)
```

Comment:
Nitpick 🔍: Short branches can be put on the same line as the condition, without braces.

Reply:
Ugh - for some reason my IDE pushes them back onto separate lines if I execute a clean up command. I'll have to Google this one so it stops happening. Thanks for pointing it out. in 2d240e8